packages/types/src/Bft.spec.js

```diff
@@ -36,7 +36,7 @@ describe('BftHashSignature', () => {
 
 describe('Justification', () => {
   const just = new Justification({
-    round_number: 32, // We get round_number from JSON, and change it to round
+    round: 32, // We get `round_number` from JSON, but both `round` and `round_number` works
```

Comment:
Out of scope here, but will make a round of these and just make sure that the all the places where we use this the `camelCase(input)` matches, so we don't have funnies like `round_number` -> `round`, rather `round_number` -> `roundNumber`. (You suggested getting rid of the maps, that will allow it - well... still needed on toJSON.)